test/integration/login.cy.ts

```diff
@@ -70,6 +70,10 @@ context('Login', () => {
         cy.login(BAD_LOGIN_AUTHENTICATION);
         toastVerifier.toastTextContains('Incorrect Username or Password');
       });
+      it('fails to authenticate an oidc user with unverified email', () => {
```

Comment:
Good catch to add this test. Is there a way to add a test to catch the double login issue? Do any of them already capture that?

Reply:
The double login should be entirely resolved now that the /okta/callback -> /okta_callback + /oidc/callback -> /oidc_callback change happened.  In my manual testing, I have never seen it show up again esp after removing the section in the authexceptionfilter where it would automatically try to do the repeated login but use the wrong url since it was only kinda written correctly for oidc.

Reply:
Amazing